include/swift/AST/DiagnosticsSema.def

```diff
@@ -1167,6 +1167,14 @@ WARNING(implementation_only_requires_library_evolution,none,
         "using '@_implementationOnly' without enabling library evolution "
         "for %0 may lead to instability during execution",
         (Identifier))
+WARNING(implementation_only_deprecated_explicit,none,
+        "'@_implementationOnly' is deprecated, use 'internal import' "
+        "and family instead",
+        ())
+WARNING(implementation_only_deprecated_implicit,none,
+        "'@_implementationOnly' is deprecated, use a bare import "
+        "as 'InternalImportsByDefault' is enabled",
```

Comment:
I would consider just having a fix-it suggesting replacing `@_implementationOnly` with `internal`, even though the attribute could be dropped instead because `InternalImportsByDefault` being enabled. It seems more straightforward and being explicit doesn't seem that harmful. This was the diagnostic can be simpler by avoiding the need to explain `InternalImportsByDefault`.

Reply:
Yeah, this will allow me to merge both diagnostics into on making the whole thing simpler. It's applied.